content/en/docs/concepts/cluster-administration/system-traces.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Traces For Kubernetes System Components
+reviewers:
+- logicalhan
+- lilic
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+System component traces record the latency of and relationships between operations in the cluster.
+
+Kubernetes components emit traces in [OTLP format](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#opentelemetry-protocol-specification) using the grpc exporter, and can be collected and routed to tracing backends using the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector#-opentelemetry-collector).
```

Comment:
```suggestion
Kubernetes components emit traces in [open telemetry format](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#opentelemetry-protocol-specification) using the gRPC exporter and can be collected and routed to tracing backends using the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector#-opentelemetry-collector).
```

Reply:
“OTLP” seems Ok to me.

Reply:
It's abbreviated from [`OpenTelemetry Protocol`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md)  and I personally prefer using long names in documentation, especially since it's a relatively new concept.

Reply:
Changed to OpenTelemetry Protocol